### DISCUSSION TITLE
I need help

### DISCUSSION THREAD

**Discussion by Beautifulcrayon** [official-stockfish/Stockfish#5398] [General]
Can you help me with chess positions please?

**Comment by Beautifulcrayon** [comment]
Stockfish

**Comment by Beautifulcrayon** [comment]
![Uploading 1DB46303-1E3A-44E7-97C5-CA973677FECA.jpeg…]()

**Comment by xu-shawn** [comment]
Check out
https://lichess.org/analysis
and
https://github.com/official-stockfish/Stockfish/wiki/Download-and-usage#free
